felles/src/main/kotlin/no/nav/emottak/util/XMLUtil.kt

```diff
@@ -14,6 +21,48 @@ import javax.xml.transform.TransformerFactory
 import javax.xml.transform.dom.DOMSource
 import javax.xml.transform.stream.StreamResult
 
+const val BIRTHDAY: Int = 6
+const val FNUMBER: Int = 11
+
+fun refParamFrikort(foresporsel: Any): String {
```

Comment:
Disse kan trekkes ut til en egen util-klasse og gis mer beskrivende navn. For eksempel til ReferenceParameterUtil, med funksjonen extractReferenceParameter(...) eller noe sånt.